tests/Unit/PointwiseFunctions/GeneralRelativity/Test_WeylPropagating.cpp

```diff
@@ -0,0 +1,328 @@
+    const tnsr::ii<DataType, SpatialDim, Frame>& ricci,
+    const tnsr::ii<DataType, SpatialDim, Frame>& extrinsic_curvature,
+    const tnsr::II<DataType, SpatialDim, Frame>& inverse_spatial_metric,
+    const tnsr::ijj<DataType, SpatialDim, Frame>& cov_deriv_extrinsic_curvature,
+    const tnsr::I<DataType, SpatialDim, Frame>& unit_interface_normal_vector,
+    const tnsr::II<DataType, SpatialDim, Frame>& projection_IJ,
+    const tnsr::ii<DataType, SpatialDim, Frame>& projection_ij,
+    const tnsr::Ij<DataType, SpatialDim, Frame>& projection_Ij) noexcept {
+  return gr::weyl_propagating<SpatialDim, Frame, DataType>(
+      ricci, extrinsic_curvature, inverse_spatial_metric,
+      cov_deriv_extrinsic_curvature, unit_interface_normal_vector,
+      projection_IJ, projection_ij, projection_Ij, -1);
+}
+
+template <size_t SpatialDim, typename DataType>
+void test_weyl_propagating(const DataType& used_for_size) {
+  {
+    tnsr::ii<DataType, SpatialDim, Frame::Inertial> (*f)(
+        const tnsr::ii<DataType, SpatialDim, Frame::Inertial>&,
+        const tnsr::ii<DataType, SpatialDim, Frame::Inertial>&,
+        const tnsr::II<DataType, SpatialDim, Frame::Inertial>&,
+        const tnsr::ijj<DataType, SpatialDim, Frame::Inertial>&,
+        const tnsr::I<DataType, SpatialDim, Frame::Inertial>&,
+        const tnsr::II<DataType, SpatialDim, Frame::Inertial>&,
+        const tnsr::ii<DataType, SpatialDim, Frame::Inertial>&,
+        const tnsr::Ij<DataType, SpatialDim, Frame::Inertial>&) =
+        &weyl_propagating_plus_wrapper<SpatialDim, Frame::Inertial, DataType>;
+    pypp::check_with_random_values<1>(f, "WeylPropagating",
+                                      "weyl_propagating_mode_plus",
+                                      {{{-1., 1.}}}, used_for_size);
+  }
+  {
+    tnsr::ii<DataType, SpatialDim, Frame::Inertial> (*f)(
+        const tnsr::ii<DataType, SpatialDim, Frame::Inertial>&,
+        const tnsr::ii<DataType, SpatialDim, Frame::Inertial>&,
+        const tnsr::II<DataType, SpatialDim, Frame::Inertial>&,
+        const tnsr::ijj<DataType, SpatialDim, Frame::Inertial>&,
+        const tnsr::I<DataType, SpatialDim, Frame::Inertial>&,
+        const tnsr::II<DataType, SpatialDim, Frame::Inertial>&,
+        const tnsr::ii<DataType, SpatialDim, Frame::Inertial>&,
+        const tnsr::Ij<DataType, SpatialDim, Frame::Inertial>&) =
+        &weyl_propagating_minus_wrapper<SpatialDim, Frame::Inertial, DataType>;
+    pypp::check_with_random_values<1>(f, "WeylPropagating",
+                                      "weyl_propagating_mode_minus",
+                                      {{{-1., 1.}}}, used_for_size);
+  }
+}
+}  // namespace
+
+namespace {
+using Affine = domain::CoordinateMaps::Affine;
+using Affine3D = domain::CoordinateMaps::ProductOf3Maps<Affine, Affine, Affine>;
+using frame = Frame::Inertial;
+constexpr size_t SpatialDim = 3;
+
+// Compare the result of gr::weyl_propagating with a calculation done
```

Comment:
I don't think this test is even useful anymore since there is a pypp test

Reply:
removed it.